main.js

```diff
@@ -26,6 +30,7 @@ function practiceGame () {
     updatePurchaseMenu ();
     clickPlus ();
     clickMinus ();
+    submitPurchase ();
     console.log('practiceGame');
 }
 
```

Comment:
Line 40 was used to debug. It should be deleted before being merged with master.